web_widget_timepicker/README.rst

```diff
@@ -83,6 +83,7 @@ Contributors
 * Michael Fried <Michael.Fried@vividlab.de>
 * Kaushal Prajapati <kbprajapati@live.com>
 * Alexandre Díaz <dev@redneboa.es>
+* David Alonso <david.alonso@solvos.es>
```

Comment:
FTR no-brainer migrations normally don't allow this.

Reply:
Thanks, I'll keep it mind in the future